modules/server/src/network/typeDefs/remoteConnections.ts

```diff
@@ -0,0 +1,16 @@
+export const RemoteConnectionNode = `#graphql
+  type Query {
+    nodes: [RemoteConnectionNode]
+  }
+
+	type RemoteConnectionNode {
+		url: String
+		name: String
+		description: String
+		documentName: String
+		availableAggregations: [String]
+		totalHits: Int
+		status: String
```

Comment:
isn't status an enum? fine to update this later.

Reply:
Just a TS type currently, not a GQL type. Will add enum for future use 👍